Examples/HackerNewsClient/.swiftlint.yml

```diff
@@ -0,0 +1,29 @@
+disabled_rules:
```

Comment:
Can you update the swiftlint file and check if you have warnings?
Use the latest version of Swiftlint (0.13.0)